article/sections/_2_graph_importance.tex

```diff
@@ -133,79 +121,21 @@ \section{Perils of disregard}
-   \item We simulate outcomes based on a chosen ``true'' multinomial logit model specification and parameters.
-   \item We estimate the specified choice model using the simulated dataset.
-   \item We use the estimated model to predict baseline probabilities using the initial covariates.
-   \item We apply the do-operator \citet{pearl_causality_2000} to our simulated data by reducing the travel distance for all individuals in the dataset.
-   This emulates a company's decision to move its employees closer to campus.
-   \item We predict new probabilities based on the estimated choice model and reduced travel distances.
-   These new probabilities are used to estimate the treatment effect under the incorrect causal graph in Figure \ref{fig:IND_GRAPH}.
-   \item We simulate new travel time and travel cost values for our commuters based on the estimated relationships from Step \ref{itm:sim-dag-estimation}.
-   This completes the effect of applying the intervention, as with a total treatment effect estimator.
-   \item We predict probabilities based on the based on the estimated choice model and altered time, distance, and cost variables.
-   These probabilities are used to estimate the treatment effect under the correct causal graph in Figure \ref{fig:DA_causal_2}.
-   \item We predict probabilities based on the chosen parameters used to generate the outcomes and based on the altered time, distance, and cost variables.
-   These probabilities are used to calculate a ``true'' treatment effect.
-   \item We repeat this process multiple times, storing the three causal effect estimates on each iteration.
-
-\end{enumerate}
-
-For our purposes, we focus on car-centric modes (Drive alone, Shared ride with another person,
-and a shared ride with two or more individuals).
-We compute the difference in mode choice probabilities from different models based on the two constructed DAGs.
 Readers interested in exploring the details of our simulation exercise can refer to our GitHub repository \citep{brathwaite_etal_2020}.
 
 
-
 We then plot histograms of the computed differences between the average probability of an individual
 in our sample choosing a car centric mode before and after implementing a policy or intervention
 aimed at reducing travel distance.
 These differences are plotted under the following assumptions: that the outcome model is the same in both scenarios and that the only difference between the two scenarios is the set of variables affected by the proposed intervention.
-This difference in the set of variables affected by the proposed intervention is a result of the two different causal graphs representing each scenario (Figures \ref{fig:IND_GRAPH} and \ref{fig:DA_causal_2}).
+This difference in the set of variables affected by the proposed intervention is a result of the two different causal graphs representing each scenario (Figures \ref{fig:IND_GRAPH}, \ref{fig:DA_causal_2}, \ref{fig:SR2_causal_2}, and \ref{fig:SR3_causal_2}).
```

Comment:
Assuming there's no reason for using SR3_causal_2, I think you meant the following:
```suggestion
This difference in the set of variables affected by the proposed intervention is a result of the two different causal graphs representing each scenario (Figures \ref{fig:IND_GRAPH}, \ref{fig:DA_causal_2}, \ref{fig:SR2_causal_2}, and \ref{fig:DA_causal_2}).
```